client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
+import {
+  Button,
+  Translation
+} from "/imports/plugins/core/ui/client/components";
+
+class LoginButtons extends Component {
+  static propTypes = {
+    capitalizeName: PropTypes.func,
+    currentView: PropTypes.string,
+    loginServices: PropTypes.func,
+    onSeparator: PropTypes.func,
+    onSocialClick: PropTypes.func
+  }
+
+  renderLoginButtons() {
+    const enabledServices = this.props.loginServices().filter((service) =>{
+      return service.enabled;
+    });
+
+    return (
+      <div>
+        {this.props.loginServices &&
+          enabledServices.map((service) => (
+            <Button
+              key={service._id}
+              className={`btn-block provider-${service.name}`}
+              primary={true}
+              bezelStyle="solid"
+              type="button"
+              data-provider={`${service.name}`}
+              onClick={() => this.props.onSocialClick(service.name)}
+            >
+              <i className={`fa fa-${service.name}`} />
+
+              {this.props.currentView === "loginFormSignInView" &&
+                <span>
+                  <Translation defaultValue="Sign in with"  i18nKey="accountsUI.signInWith" />
+                </span>
+              }
+              {this.props.currentView === "loginFormSignUpView" &&
+                <span>
+                  <Translation defaultValue="Sign up with" i18nKey="accountsUI.signUpWith" />
+                </span>
+              }
+
+              &nbsp;<span>
+                      <Translation defaultValue={this.props.capitalizeName(service.name)} i18nKey={`social.${service.name}`} />
+                    </span>
+          </Button>
+          ))
+        }
+      </div>
+    );
+  }
+
+  renderSeparator() {
```

Comment:
You can render words as well with `<Divider label="or" />`
@rymorgan what do you think, this is more a design question than anything else